### ISSUE TITLE
this could support ios applications?

### ISSUE THREAD

**Issue by arden** [IBM-Swift/Aphid#1]
this could support ios applications?

**Comment by rfdickerson** [comment]
In theory, yes- the BlueSocket library should work on iOS. However, we have not tested it yet. If you do get the chance, please let us know if it works on iOS. We are currently working to move this library from 6-06 SNAPSHOT to the nightlies which had the great API name changes.

```
ErrorProtocol -> Error
NSData -> Data
```

etc.

**Comment by billabt** [comment]
There's no good reason why BlueSocket shouldn't work on iOS.  However, last time I tried to build an iOS app using Swift 3, Xcode couldn't do it.  It's been awhile since I tried so it'll probably be worth at least an attempt at it with Xcode 8 Beta 3 using the 7/25 toolchain.  Will advise.

P.S.  The switch from NSData to Data hasn't happened yet since there's still some serious incompatibilities between the implementation on macOS and on Linux.  I'm waiting for those to be fixed before moving to Data.
